erp-playbook/library/mr_provisioner_power.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/python
+
+import json
+import requests
+import time
+from datetime import datetime, timedelta
+
+from future.standard_library import install_aliases
+install_aliases()
+
+from urllib.parse import urljoin  # nopep8
+from urllib import quote  # nopep8
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: mr-provisioner-power
+
+short_description: Wait for power on/off.
+
+description:
+    - Wait for power on/off.
+    - Suppot to force power on/off when timeout reached.
+
+options:
+    machine_name:
+        description: Machine name
+        required: true
+    url:
+        description: url to provisioner instance in the form of http://172.27.80.1:5000/
+        required: true
+    token:
+        description: Mr. Provisioner auth token
+        required: true
+    wait_for_power:
+        description: Desired power state
```

Comment:
It would be good to provide the possible values for this parameter (i think [off|on]),

Reply:
Yeah, done.